include/seqan3/alignment/all.hpp

```diff
@@ -239,8 +238,7 @@
  *
  * The edit configuration can be further specialised with following configs:
  *  * Set a minimal score, i.e. specify the seqan3::align_cfg::min_score configuration
- *  * Compute a semi-global alignment where the first sequence is contained in the second sequence and vice versa but
- *    not any arbitrary free end-gap configuration.
+ *  * Compute a semi-global alignment where either the end-gaps of the first sequence are free or the end-gaps of the second sequence.
```

Comment:
This sentence is currently not true;

```c++
seqan3::align_cfg::free_end_gaps_sequence1_leading{true},
seqan3::align_cfg::free_end_gaps_sequence2_leading{false},
seqan3::align_cfg::free_end_gaps_sequence1_trailing{true},
seqan3::align_cfg::free_end_gaps_sequence2_trailing{false}
```

Is the only accepted configuration.

Reply:
Ok, I thought you said you could run both versions. But this is not implemented yet?

Reply:
Right :)

I'm working out the details to make his happen :)